### PR TITLE
commit update Sys.Date()-1 to "Sys.Date() (row 3, 21,32,83. and Row30…

### PR DESCRIPTION
**Pull request by jeannewu**
…74 "add_indent(c(1:8)) to update as "add_indent(c(1:9))

### CODE REVIEW

File: .Rhistory
Comment (jacobmpeters) [inline .Rhistory:1]: Remove .Rhistory from the commit.

File: weekly_ccc_report_gcp_pipeline.Rproj
Comment (jacobmpeters) [inline weekly_ccc_report_gcp_pipeline.Rproj:1]: Remove the Rproj file from the commit and add "*.Rproj" to your .gitignore file.

File: stg/.DS_Store
Comment (jacobmpeters) [inline stg/.DS_Store:1]: Remove .DS_Store from your commit and add it to your .gitignore file.

File: prod/Consolidated_Weekly_Report_4Pipeline_BQupdated_06082023.Rmd
Comment (jacobmpeters) [inline prod/Consolidated_Weekly_Report_4Pipeline_BQupdated_06082023.Rmd:1]: I don't see anything that will affect the cloud run.

**Review by jacobmpeters — CHANGES_REQUESTED** [review]

**Comment by jacobmpeters** [comment]
I closed this pull request. I'll try to merge your most recent commit as you requested on Teams.

**Comment by jeannewu** [comment]
Thanks a lot, Jake

**Pull request closed without merging**